server/index.js

```diff
@@ -15,5 +16,4 @@ app.listen(port, () => {
 
 app.get('/host', (req, res) => {
   SingleOwner.find().then(data => res.send(data));
```

Comment:
@SimonC989 @carl12 `.then` should go on a new line. Be sure to handle errors with a `.catch`

Reply:
@SimonC989 @carl12 `data` is not a good, descriptive variable name